tensorflow_app/views/export_graphdef.py

```diff
@@ -1,58 +1,196 @@
-from django.views.decorators.csrf import csrf_exempt
-from django.http import JsonResponse
-import yaml
-from datetime import datetime
+import os
 import random
 import string
-import os
-from ide.utils.jsonToPrototxt import json_to_prototxt
+import yaml
+
+from datetime import datetime
+from django.views.decorators.csrf import csrf_exempt
+from django.http import JsonResponse
+from keras.models import Model
+from keras.models import model_from_json
 import tensorflow as tf
-import sys
-BASE_DIR = os.path.dirname(os.path.dirname(
-    os.path.dirname(os.path.abspath(__file__))))
-sys.path.insert(0, BASE_DIR + '/media/')
-sys.path.insert(0, BASE_DIR + '/tensorflow_app/caffe-tensorflow/')
+from keras import backend as K
+from keras_app.views.layers_export import data, convolution, deconvolution, pooling, dense, dropout, embed,\
+    recurrent, batch_norm, activation, flatten, reshape, eltwise, concat, upsample, locally_connected,\
+    permute, repeat_vector, regularization, masking, gaussian_noise, gaussian_dropout, alpha_dropout
+BASE_DIR = os.path.dirname(
+    os.path.dirname(
+        os.path.dirname(
+            os.path.abspath(__file__))))
 
 
 def randomword(length):
     return ''.join(random.choice(string.lowercase) for i in range(length))
 
 
+def json2pbtxt(json_string, f, output_fld):
+    net_model = model_from_json(json_string)
+    num_output = 1
```

Comment:
Why create this variable if it is hardcoded?

Reply:
Yeah, this was carried over from my script.